service-template/README.md

```diff
@@ -21,3 +21,5 @@ Template for blockscout Rust service
 🔧   Initializing a fresh Git repository
 ✨   Done! New project created /Users/levlymarenko/poa/blockscout-rs/test-project
 ```
+
+3. Go into created service directories and recursively rename all `_Cargo.toml` to `Cargo.toml`
```

Comment:
just use post hook of `cargo-generate` to automate this:

https://cargo-generate.github.io/cargo-generate/templates/scripting.hook-types.html#post